README.md

```diff
@@ -106,140 +106,6 @@ If both values are invalid semver values, then the values are compared alphabeti
 | versionA | <code>string</code> \| <code>null</code> \| <code>undefined</code> | The first version to compare |
 | versionB | <code>string</code> \| <code>null</code> \| <code>undefined</code> | The second version to compare |
 
-**Example**  
```

Comment:
This appears to have removed all the examples from the README?

Reply:
WAAAAH